stonesoup/plotter.py

```diff
@@ -271,6 +275,32 @@ def plot_tracks(self, tracks, mapping, uncertainty=False, particle=False, track_
         else:
             self.ax.legend(handles=self.handles_list, labels=self.labels_list)
 
+    def plot_density(self, state_sequences: Sequence[StateMutableSequence], index=-1, mapping=(0, 2), n_bins=300,
```

Comment:
Could a docstring be added here?

Reply:
Will do.
My IDE had set the limit to 120 for some reason. Does the 100 character limit apply to the example as well?

Reply:
> Does the 100 character limit apply to the example as well?

It does, but we're a bit more relaxed about  examples, as long as they look okay when rendered.